lib/SILOptimizer/Mandatory/PerformanceDiagnostics.cpp

```diff
@@ -631,7 +631,7 @@ bool PerformanceDiagnostics::visitInst(SILInstruction *inst,
 
     switch (inst->getKind()) {
     case SILInstructionKind::BeginApplyInst:
-      // Not all begin_applys necessarily allocate. But it's difficult to
+      // Not all begin_applies necessarily allocate. But it's difficult to
```

Comment:
This change is wrong. It really should be `begin_applys`. _Maybe_ you could justify quotes around `begin_apply`, ala `'begin_apply's`, but the original text was correct.

Reply:
reverted